specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json

```diff
@@ -106,41 +106,1303 @@
+        "x-ms-examples": {
+          "Inconsistency detect example": {
+            "$ref": "./examples/InconsistencyDetect.json"
+          }
+        }
+      }
+    },
+    "/timeseriesgroups/{timeSeriesGroupId}/inconsistency/query": {
+      "post": {
+        "summary": "Query time series properties from a group of similar time series.",
+        "description": "This operation helps query properties of one group, for example the inconsistent series among a group series with similar trend.",
+        "operationId": "TimeSeriesGroup_InconsistencyQuery",
+        "parameters": [
+          {
+            "$ref": "#/parameters/timeSeriesGroupId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Start time and end time is necessary.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/InconsistencyQueryRequest"
+            }
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation.",
+            "schema": {
+              "$ref": "#/definitions/InconsistencyQueryResponse"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Inconsistency query example": {
+            "$ref": "./examples/InconsistencyQuery.json"
+          }
+        }
+      }
+    },
+    "/timeseriesgroups/{timeSeriesGroupId}/timeseries": {
+      "get": {
+        "summary": "List TimeSeries that belongs to a TimeSeriesGroup.",
+        "description": "List TimeSeries that belongs to a TimeSeriesGroup. One TimeSeriesGroup could have multiple TimeSeries.",
+        "operationId": "TimeSeriesGroup_ListSeries",
+        "parameters": [
+          {
+            "$ref": "#/parameters/timeSeriesGroupId"
+          },
+          {
+            "$ref": "#/parameters/next"
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation.",
+            "schema": {
+              "$ref": "#/definitions/TimeSeriesList"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get TimeSeries of a TimeSeriesGroup" : {
+            "$ref": "./examples/TimeSeriesGroupListTimeSeries.json"
+          }
+        }
+      }
+    },
+    "/timeseriesgroups/{timeSeriesGroupId}/timeseries/{timeSeriesId}": {
+      "post": {
```

Comment:
Not the right way to represent relation between time series group and time series

Reply:
Resolved, Removed group related operations